projects/magento/tests/admin/RequiredSettingsAutoConfig.spec.js

```diff
@@ -44,4 +44,23 @@ test.describe('Configure required settings', () => {
         await adminSection.waitForPageLoad(page);
         await expect(page.locator('.message-success', { has: page.locator("//div[text()='You saved the configuration.']")})).toBeVisible();
     });
+
+    test('auto mode fails with bad api key', async ({ page }) => {
+        page.on('dialog', dialog => dialog.accept());
+        const requiredSettingsHeader = page.locator("#payment_us_adyen_group_all_in_one_adyen_required_settings-head");
```

Comment:
It would be nice to extract the repeated parts to a function and call it in both tests

Reply:
Done :)